packages/ember-data/lib/system/store/serializer-response.js

```diff
@@ -3,6 +3,35 @@ import Model from 'ember-data/system/model/model';
 const get = Ember.get;
 
 /**
+  This is a helper method that validates a JSON API top-level document
+
+  The format of a document is described here:
+  http://jsonapi.org/format/#document-top-level
+
+  @method isValidJsonApiDocument
```

Comment:
This doesn't match the name of the function.

Do you also think that "JsonApi" is implicit? Maybe `validateDocument()` would be sufficient?